src/main/java/io/mockify/hoster/dao/FileRepository.java

```diff
@@ -97,13 +97,13 @@ public void saveHtml(String html , Project project, String userId){
         try {
             file.createNewFile();
         } catch (IOException e) {
-            e.printStackTrace();
+            log.error("Couldn't create html file",e);
```

Comment:
I would suppose that file creation failure should propogate error, as there's no indicator for outside caller to know how to treat further execution. The API of FileRepo should be adjusted accordingly.